.github/workflows/black_and_test.yml

```diff
@@ -18,7 +18,13 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        python setup.py install
         pip install black
+        pip install pytest
 
     - name: Black
       run: black . --check
+
+    - name: Test with pytest
+      run: |
+        pytest
```

Comment:
Following [`pytest`'s best practices for integration with `setuptools`](http://doc.pytest.org/en/3.0.2/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner), I prefer this to be `python setup.py test` (let's keep the standards, yay, hahaha). Also there's no need for the pipe is there's only one command/line: 

```yaml
    - name: Tests
       run: python setup.py test
```